.github/workflows/python-bindings-publish.yml

```diff
@@ -3,9 +3,8 @@ name: Build and Upload Wheels for the Python Bindings
 on:
   push:
     branches:
-      # disabled until python binding is updated
-      # - develop
-      - dev
+      - develop
```

Comment:
Not sure if this should be done like this now because it could interfere with the production builds? @bingyanglin do you know if it would be fine?

Reply:
It won't interfere with production unless develop is merged into production, and when that happens, CI should pass anyway.

Reply:
It should be fine.